### PR TITLE
[1LP][RFR] adding search support to entities.get_entity()

### PR DESCRIPTION
**Pull request by izapolsk**
This PR doesn't add any search reset functionality. 
test owner should take of it by himself. 
For example add view.entities.search.clear_search() to nav destination->resetter()

### CODE REVIEW

File: widgetastic_manageiq/__init__.py [hunk 1]
```diff
@@ -2614,14 +2614,18 @@ def get_all(self, surf_pages=False):
                                 for el in self._current_page_elements])
             return entities
 
-    def get_entity(self, surf_pages=False, **keys):
+    def get_entity(self, surf_pages=False, use_search=False, **keys):
```
Comment (quarckster) [inline widgetastic_manageiq/__init__.py]: **REQUIRED**
Please add `use_search` argument to the docstring.

File: cfme/common/vm.py [hunk 2]
```diff
@@ -325,11 +325,8 @@ def find_quadicon(self, from_any_provider=False, use_search=True):
         if 'Grid View' != view.toolbar.view_selector.selected:
             view.toolbar.view_selector.select('Grid View')
 
-        if use_search:
-            search.normal_search(self.name)
-
         try:
-            return view.entities.get_entity(name=self.name, surf_pages=True)
+            return view.entities.get_entity(name=self.name, surf_pages=True, use_search=True)
```
Comment (mshriver) [inline cfme/common/vm.py]: **Required/Question**

Why pass `True` instead of `use_search` that was passed into find_quadicon?

Comment (izapolsk) [reply cfme/common/vm.py]: it doesn't have anything behind except sleeplessness :). fixed

File: widgetastic_manageiq/__init__.py (hunk 1 above)
Comment (izapolsk) [reply widgetastic_manageiq/__init__.py]: done

**Comment by izapolsk** [comment]
PRT errors aren't related to my changes

**Pull request merged**